pages/common/wp.md

```diff
@@ -11,6 +11,14 @@
 
 `wp cli update`
 
+- Download a fresh WordPress installation to current directory:
+
+`wp core download [--locale=<locale>]`
```

Comment:
We don't really have syntax for optional arguments, but we can mention it in the description. Also, we use token syntax for everything which the user has to input:
```suggestion
- Download a fresh WordPress installation to current directory, optionally specifying the locale:

`wp core download --locale={{locale}}`
```

Reply:
Ah, okay. Yeah that works fine for me 👍